deepmd/entrypoints/train.py

```diff
@@ -112,9 +125,17 @@ def _do_work(jdata: Dict[str, Any], run_opt: RunOptions):
         reset_default_tf_session_config(cpu_only=True)
 
     # init the model
-    model = DPTrainer(jdata, run_opt=run_opt)
-    rcut = model.model.get_rcut()
-    type_map = model.model.get_type_map()
+    if not run_opt.multi_task:
+        model = DPTrainer(jdata, run_opt=run_opt)
+        rcut = model.model.get_rcut()
+        type_map = model.model.get_type_map()
+    else:
+        model = DPTrainer_mt(jdata, run_opt=run_opt)
+        for model_name in model.model_dict.keys():
```

Comment:
Maybe we need some check on `rcut` and `type_map` among sub models. Otherwise, this code segment will only use those of the last sub mdoel.

Reply:
type_map is consistent now, however rcut may vary between different descriptors, so I guess I'll put a list through building the data docker